wakuv2/waku.go

```diff
@@ -1043,23 +1047,30 @@ func (w *Waku) query(ctx context.Context, peerID peer.ID, pubsubTopic string, to
 		PubsubTopic:   pubsubTopic,
 	}
 
+	cursorLogField := zap.Skip()
```

Comment:
Why not just log it as `nil`?

Reply:
To not even add the `cursor=nil` message in the log line!

Reply:
I just think it's ok to have it there, it doesn't hurt o have it explicit.
But it hurts to add this redundant `if`s in the code IMO 🙂

Reply:
hm. I agree with you.
I reverted that part of the code and log the cursor as a Stringer just to avoid having this `if`